kafkaconnect/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package kafkaconnect contains utility functions and structures for processing
+// Kafka Connect compatible data.
+package kafkaconnect
```

Comment:
the definition of nitpick 👀
```suggestion
// Kafka Connect compatible data.

package kafkaconnect
```

Reply:
This is intentional, it's a godoc comment that will show up in our documentation. More info about [godoc comments](https://tip.golang.org/doc/comment).